test/test_embedded.py

```diff
@@ -21,10 +21,12 @@
 if platform != "linux" and platform != "darwin":
     pytest.skip("Currently only supported on linux", allow_module_level=True)
 
+GRPC_EMBEDDED_DEFAULT_PORT = 50060
+
 
 def test_embedded__init__(tmp_path):
     assert (
-        EmbeddedDB(EmbeddedOptions(port=6666, persistence_data_path=tmp_path)).options.port == 6666
+        EmbeddedDB(EmbeddedOptions(port=8079, persistence_data_path=tmp_path)).options.port == 8079
```

Comment:
you can make `EMBEDDED_DEFAULT_PORT = 8079` and use it here. I would then also rename `GRPC_EMBEDDED_DEFAULT_PORT` to `EMBEDDED_DEFAULT_GRPC_PORT`